pallet/marketplace/src/lib.rs

```diff
@@ -325,6 +325,8 @@ pub mod pallet {
 		TokenOnAuction,
 		/// No tokens were specified in the listing
 		EmptyTokens,
+		/// Token does not have an owner, likely the token has been burnt
+		NoTokenOwner,
```

Comment:
nit, NoToken may be clearer and align with the same error in other pallets
```suggestion
		/// The token does not exist
		NoToken,
```